### PR TITLE
Catch IOError

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

Fixes:

image

### CODE REVIEW

File: medusa/helpers.py [hunk 1]
```diff
@@ -1040,7 +1040,11 @@ def backupConfigZip(fileList, archive, arcname=None):
     try:
         a = zipfile.ZipFile(archive, 'w', zipfile.ZIP_DEFLATED, allowZip64=True)
         for f in fileList:
-            a.write(f, os.path.relpath(f, arcname))
+            try:
```
Comment (ratoaq2) [inline medusa/helpers.py]: You will fail the backup and the method will still return true

**Comment by medariox** [comment]
I propose we don't let Medusa boot if free space < 50 MB! 😈

**Comment by fernandog** [comment]
can we close this?

**Comment by duramato** [comment]
This is still valid, fixes what it proposes to

**Comment by fernandog** [comment]
as discussed

**Pull request closed without merging**